### PR TITLE
Task force meetings page

### PR DESCRIPTION
**Pull request by navneetjoshi223**

### CODE REVIEW

**Review by anaximander — CHANGES_REQUESTED** [review]
Overall looking pretty good, most changes I've requested are just removing unused code. I have some UI/UX feedback that send you directly in Slack.

File: src/js/components/TaskForceMeetings.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    fieldMapping: {
+      date: "Meeting Date",
+      summary: "Summary",
+      attachments: "Attachments",
+    },
+  });
+
+  console.log(taskForceMeetings);
+
+  // const columns = useMemo(
```
[the remark this answers is not in the document]
Comment (navneetjoshi223) [reply src/js/components/TaskForceMeetings.jsx]: Removed it

File: src/js/components/TaskForceMeetings.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import { useMemo } from "react";
+import useAirtableCMS from "../hooks/useAirtableCMS";
+import Markdown from "react-markdown";
+import { useTable } from "react-table";
```
[the remark this answers is not in the document]
Comment (navneetjoshi223) [reply src/js/components/TaskForceMeetings.jsx]: yes I was experimenting with it. Removed the import since we're not using it!

**Review by anaximander — APPROVED** [review]
LGTM 👍 We can do another pass for styling later.

**Pull request merged**